### ISSUE TITLE
Rule doesn't apply when sql is multiline string

### ISSUE THREAD

**Issue by rkuska** [returntocorp/sgrep-rules#2755]
**Describe the bug**
When I edit the example for go tainted-sql-string rule to make the sql strings to start at new line the rule doesn't report the concat of sql strings.

**To Reproduce**
1. Go to https://semgrep.dev/playground/r/go.aws-lambda.security.tainted-sql-string.tainted-sql-string?editorMode=advanced
2. Change `  _, err = db.Exec("DELETE FROM table WHERE Id = " + request.Get("Id"))` to multiline string:
 ```go
  _, err = db.Exec(`
  DELETE FROM table WHERE Id = ` + request.Get("Id"))
```
4. Run the rule
5. SQL is no longer reported

**Expected behavior**
I would expect the string to be still reported.

**Comment by LewisArdern** [comment]
Good catch @rkuska I updated the regex to consider whitespace.
